src/Components/Play/HumanVsHuman.jsx

```diff
@@ -396,83 +400,115 @@ class HumanVsHuman extends Component {
-
-        // Construct a HTTP POST query
-        var query = new FormData();
-        query.set('game_id', gameid);
-        query.set('user_id', userid);
-
-        // If the user clicks anywhere outside the box then close it and decline offer
-        window.onclick = (event) => {
-            if (event.target === confBox) {
-                // Decline offer
-                query.set('response', 'false');
-
-                // Send the POST request to the server
-                server.post('/respondoffer', query)
-                    .then(function(response) {
-                        // Close offer window
-                        confBox.style.display = "none";
-                    })
-                    .catch(function(error) {
-                        var tmp = $('<div></div>');
-                        tmp.html(error.response.data);
-
-                        var message = $('p', tmp).text();
-                        console.log(message);
-                    });
-            }
-        }
-
-        // Accept the draw offer
-        const acceptDrawBtn = document.getElementById("accept-draw");
-        acceptDrawBtn.onclick = (event) => {
-            query.set('response', 'true');
-
-            // Send the POST request to the server
-            server.post('/respondoffer', query)
+        if (userid !== draw_offer_user_id) {
+            axios.get(`https://negativei2-server.herokuapp.com/getgame/${gameid}`)
                 .then(function(response) {
-                    // Close offer window
-                    confBox.style.display = "none";
-
-                    // Go to end game page
-                    //this.props.history.push('/endgame');
-                })
-                .catch(function(error) {
-                    var tmp = $('<div></div>');
-                    tmp.html(error.response.data);
+                    if (userid === response.data.players.b || userid === response.data.players.w) {
+                        // User is opponent not spectator
+                        const confBox = document.getElementById("draw-received-confirmation");
+
+                        // Open the confirmation window
+                        confBox.style.display = "block";
+
+                        // Construct a HTTP POST query
+                        var query = new FormData();
+                        query.set('game_id', gameid);
+                        query.set('user_id', userid);
+
+                        // If the user clicks anywhere outside the box then close it and decline offer
+                        window.onclick = (event) => {
+                            if (event.target === confBox) {
+                                // Decline offer
+                                query.set('response', 'false');
+
+                                // Send the POST request to the server
+                                axios.post('https://negativei2-server.herokuapp.com/respondoffer', query)
```

Comment:
Should use `server.post('/respondoffer', query)` here instead of `axios`. _See #110._